deploy/srep-1000-ethtool-exporter/09-DaemonSet.yaml

```diff
@@ -0,0 +1,190 @@
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
I presume most of this is copied from the existing node-exporter DS in an OCP cluster? Might be good to (if the list is short) add a comment to the file with what you have modified/changed.